.github/workflows/smoke.yaml

```diff
@@ -13,47 +13,20 @@ jobs:
       matrix:
         # pylibjuju does not currently support Juju 4.x
         # The smoke tests do not yet work on Juju 2.9.
-        juju-version: ['3.5']
-        charmcraft-version: ['2.x', '3.x']
-        cloud: ['lxd', 'microk8s']
+        juju-channel: ['3/stable']
+        charmcraft-channel: ['2.x/stable', '3.x/stable']
+        preset: ['machine', 'microk8s']
 
     env:
-      JUJU_VERSION: "${{ matrix.juju-version }}"
+      CONCIERGE_JUJU_CHANNEL: "${{ matrix.juju-channel }}"
+      CONCIERGE_CHARMCRAFT_CHANNEL: "${{ matrix.charmcraft-channel }}"
```

Comment:
I figure that converting these to explicit command line arguments to concierge would be cleaner, more targeted and it makes the whole env block redundant.

Reply:
The env block felt clean to me, but no objection to switching it over.